services/shotgrid_common/utils.py

```diff
@@ -672,6 +679,24 @@ def get_sg_entities(
 
                 parent_id = cat_ent_name
 
+            # Transform task_assignees list of dictionary entries
+            # to just a list of the login names as used in AYON DB
+            # so it's easier later to set
+            task_assignees = sg_entity.get("task_assignees")
+            if task_assignees:
+                task_assignees_list = []
+                for assignee in task_assignees:
+                    # Skip task assignments that aren't from a human user (i.e. groups)
+                    # TODO: add support for group assignments
+                    if assignee["type"] != "HumanUser":
+                        continue
+                    sg_user = get_sg_user_by_id(
+                        sg_session, assignee["id"], extra_fields=["login"]
+                    )
+                    task_assignees_list.append(sg_user["login"])
```

Comment:
Here we will need to refactor the code a little so we are actually getting AY user names based on `sg_user_id` key store in AY user entity.data. Look here #76 .